lib/core/package.json

```diff
@@ -121,6 +121,7 @@
     "shelljs": "^0.8.4",
     "stable": "^0.1.8",
     "style-loader": "^1.2.1",
+    "telejson": "^5.0.2",
```

Comment:
do we still need this? are we still stringifying anything?

Reply:
Yes we are